src/i18n/data/countries.json

```diff
@@ -0,0 +1,1501 @@
+[
+    {"code": "AD",
+      "name": "Andorra",
+      "native": "Andorra",
+      "capital": "Andorra la Vella",
+      "currency": ["EUR"],
+      "languages": ["ca"]},
+    {"code": "AE",
+      "name": "United Arab Emirates",
+      "native": "دولة الإمارات العربية المتحدة",
+      "capital": "Abu Dhabi",
+      "currency": ["AED"],
+      "languages": ["ar"]},
+    {"code": "AF",
+      "name": "Afghanistan",
+      "native": "افغانستان",
+      "capital": "Kabul",
+      "currency": ["AFN"],
+      "languages": ["ps", "uz", "tk"]},
+    {"code": "AG",
+      "name": "Antigua and Barbuda",
+      "native": "Antigua and Barbuda",
+      "capital": "Saint John's",
+      "currency": ["XCD"],
+      "languages": ["en"]},
+    {"code": "AI",
+      "name": "Anguilla",
+      "native": "Anguilla",
+      "capital": "The Valley",
+      "currency": ["XCD"],
+      "languages": ["en"]},
+    {"code": "AL",
+      "name": "Albania",
+      "native": "Shqipëria",
+      "capital": "Tirana",
+      "currency": ["ALL"],
+      "languages": ["sq"]},
+    {"code": "AM",
+      "name": "Armenia",
+      "native": "Հայաստան",
+      "capital": "Yerevan",
+      "currency": ["AMD"],
+      "languages": ["hy", "ru"]},
+    {"code": "AO",
+      "name": "Angola",
+      "native": "Angola",
+      "capital": "Luanda",
+      "currency": ["AOA"],
+      "languages": ["pt"]},
+    {"code": "AQ",
+      "name": "Antarctica",
+      "native": "Antarctica",
+      "capital": "",
+      "currency": [],
+      "languages": []},
+    {"code": "AR",
+      "name": "Argentina",
+      "native": "Argentina",
+      "capital": "Buenos Aires",
+      "currency": ["ARS"],
+      "languages": ["es", "gn"]},
+    {"code": "AS",
+      "name": "American Samoa",
+      "native": "American Samoa",
+      "capital": "Pago Pago",
+      "currency": ["USD"],
+      "languages": ["en", "sm"]},
+    {"code": "AT",
+      "name": "Austria",
+      "native": "Österreich",
+      "capital": "Vienna",
+      "currency": ["EUR"],
+      "languages": ["de"]},
+    {"code": "AU",
+      "name": "Australia",
+      "native": "Australia",
+      "capital": "Canberra",
+      "currency": ["AUD"],
+      "languages": ["en"]},
```

Comment:
`en_AU` instead ?

Reply:
We should keep it like this here. If we later want to use the country/language relationship in our app, I would add some logic to:
- check if a locale with country code exists
- if not then get the locale without country code

I am going to use this logic to display the country languages first in the language dropdown in a future PR